### PR TITLE
Beginning of Voxel engine

### PR DESCRIPTION
**Pull request by waredavid**

### CODE REVIEW

File: CMakeLists.txt.orig [hunk 1]
```diff
@@ -0,0 +1,93 @@
+cmake_minimum_required(VERSION 2.8)
```
Comment (ostlerc) [inline CMakeLists.txt.orig]: Is there a reason we want to keep this orig file?

File: client/src/voxel_rendering/block.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#include "block.h"
```
Comment (ostlerc) [inline client/src/voxel_rendering/block.cpp]: What do we plan to do with this block class? It just seems like an over glorified boolean at this point.

File: CMakeLists.txt.orig (hunk 1 above)
Comment (ksundberg) [inline CMakeLists.txt.orig]: No, it can be removed

File: client/src/voxel_rendering/block.cpp (hunk 2 above)
Comment (ksundberg) [inline client/src/voxel_rendering/block.cpp]: From their proposal this is a place holder to enable culling in the GUI

**Pull request merged**